.github/workflows/ci-daily.yml

```
@@ -0,0 +1,98 @@
+name: Daily Chromatic test
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

That's midnight for EU but not US :wink: 
```suggestion
    - cron: "0 6 * * *"
```